docs/source/markdown/options/hostname.pod.md

```diff
@@ -4,4 +4,7 @@
 ####> are applicable to all of those.
 #### **--hostname**=*name*
 
-Set a hostname to the pod.
+Sets the pod's hostname inside all containers.
```

Comment:
I haven't looked at all options, but from a quick overview I find a strong convention of using imperative, not declarative. "Set", not "Sets". I tentatively think we should stick with that convention.